Explorer/Assets/DCL/Web3Authentication/FakeWeb3Authenticator.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DCL.Web3Authentication
+{
+    public class FakeWeb3Authenticator : IWeb3Authenticator
+    {
+        public async Task<(IWeb3Identity identity, IWeb3Identity ephemeralIdentity)> Login(CancellationToken cancellationToken) =>
```

Comment:
We currently got no way of knowing if this makes any sense, right?

Reply:
Refactor so it returns an identity instead, which should be sufficient for the rest of the operations